packages/visualisation/src/components/KommunStatisticsBox/index.js

```diff
@@ -27,52 +27,77 @@ const GridWrapper = styled.div`
   margin-top: 1.5rem;
 `
 
-const KommunStatisticsBox = ({
-  name,
-  totalBookings,
-  totalCars,
-  totalCargo,
-  totalCo2,
-  totalCapacity,
-  averageDeliveryTime,
-  averageCost,
-  totalDelivered,
-  totalQueued,
-  averageQueued,
-  averageUtilization,
-}) => {
+const getTitle = (key) => {
+  switch (key) {
+    case 'totalCars':
+      return 'Antal fordon'
+    case 'totalCo2':
+      return 'co2'
```

Comment:
```suggestion
      return 'CO₂'
```

Eller prova med någon av dessa:

- UNICODE `U+02082`
- HEX CODE `&#x2082;`
- HTML CODE `&#8322;`